### ISSUE TITLE
Better GraphQL -> SQL

### ISSUE THREAD

**Issue by orefalo** [rexxars/sql-to-graphql#51]
What about the reverse?

Back to the old ORM days, going DB->Object never worked 100%
However, going Object -> DB schema always does.

**Comment by tevaum** [comment]
I agree with @orefalo. Is there any interest in implementing this approach? Or any other tool that does that?
